CONTRIBUTING.rst

```diff
@@ -194,6 +194,27 @@ title                                  string
 
 For a full schema of a category JSON object, please see https://github.com/pyvideo/data/blob/master/.schemas/category.json
 
+Creating Data From Youtube Channel
+~~~~~~~~~~~~
+
+You can easily create basic data from already created youtube channel
+
+#. install requirements::
+
+   $ pip install -r tools/requirements.txt
+
+#. get youtube api key from google console, please see https://developers.google.com/youtube/registering_an_application
+#. set GOOGLE_API_KEY environment::
+
+   $ export GOOGLE_API_KEY=what_you_get_from_google_console
+
+#. run ./tools/youtube.py -l channe list # ( in this case pycon jp 2015 )::
```

Comment:
This is confusing. Maybe something like

Run `tools/youtube.py -l ####`, where `####` is the playlist ID. For example, run

    $ python tools/youtube.py -l PLMkWB0UjwFGkE3BIm4qpwaK5oPrmLLJoT

to retrieve the PyCon JP 2015 data.